action.yml

```diff
@@ -105,11 +105,12 @@ runs:
         GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         # https://docs.github.com/en/rest/branches/branches#list-branches
+        # branches are sorted ASC, so oldest branches will be first
         RESP_CODE=$(curl -w %{http_code} -s -L -o __branches.json \
           -H "Accept: application/vnd.github+json" \
           -H "Authorization: Bearer ${{ github.token }}" \
           -H "X-GitHub-Api-Version: 2022-11-28" \
-          https://api.github.com/repos/$GITHUB_REPOSITORY/branches \
+          https://api.github.com/repos/$GITHUB_REPOSITORY/branches?per_page=100 \
```

Comment:
This is the main fix, the default is 30 records, which means there aren't enough for the calculations later on in this file and it ends up thinking that CMS 4 is the previous major branch